src/components/molecules/timeline-events/order-edit/declined.tsx

```diff
@@ -1,25 +1,44 @@
-import { useAdminCustomer } from "medusa-react"
+import { useAdminCustomer, useAdminOrderEdit, useAdminUser } from "medusa-react"
 import React from "react"
-import { ByLine } from "."
+
 import { OrderEditEvent } from "../../../../hooks/use-build-timeline"
 import XCircleIcon from "../../../fundamentals/icons/x-circle-icon"
 import EventContainer from "../event-container"
+import { ByLine } from "."
 
 type EditDeclinedProps = {
   event: OrderEditEvent
 }
 
 const EditDeclined: React.FC<EditDeclinedProps> = ({ event }) => {
-  const { customer } = useAdminCustomer(event.edit.declined_by as string)
+  const { order_edit: orderEdit } = useAdminOrderEdit(event.edit.id)
+
+  const declinedByAdmin = event.edit.declined_by?.startsWith("usr")
+
+  const { user } = useAdminUser(event.edit.declined_by as string, {
```

Comment:
suggestion: Since declined_by can be undefined, here the request could pass an undefined id isn't it? should we do that only if the value exists. the risk is that the server throws on invalid id instead of returning a random user ahah

Reply:
the request won't fire if `!!event.edit.declined_by === false` (the `enabled` prop on the line below)

Reply:
from the jsdoc it says that the enabled only prevent from refetching no? `Set this to `false` to disable automatic refetching when the query mounts or changes query keys.` So I thought the first one would be fired but none after my bad 😂